admin/sql/create_db.sql

```diff
@@ -2,5 +2,5 @@
 
 -- Create the user and the database. Must run as user postgres.
 
-CREATE USER acousticbrainz NOCREATEDB NOCREATEUSER;
-CREATE DATABASE acousticbrainz WITH OWNER = acousticbrainz TEMPLATE template0 ENCODING = 'UNICODE';
+-- CREATE USER acousticbrainz NOCREATEDB NOCREATEUSER;
+CREATE DATABASE acousticbrainz94 WITH OWNER = acousticbrainz TEMPLATE template0 ENCODING = 'UNICODE';
```

Comment:
this was for my testing. On spike we'll be running on 2 different database servers so the database name can be the same. We'll have to sort out a step by step process.